azure_arc_k8s_jumpstart/aro/dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM mcr.microsoft.com/azure-cli:latest
+FROM mcr.microsoft.com/azure-cli:2.18.0
 LABEL Ali Hussain "alhussai@microsoft.com"
 
 ADD ./ /root/
-RUN az extension add -n aro --index https://az.aroapp.io/stable \
-    && az extension add --name connectedk8s
+RUN az extension add --name connectedk8s
 RUN wget -q https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz -O openshift-client-linux.tar.gz \
     && tar -zxvf openshift-client-linux.tar.gz -C /bin/
 RUN wget -q https://get.helm.sh/helm-v3.2.2-linux-amd64.tar.gz -O helm-v3.2.2-linux-amd64.tar.gz \
```

Comment:
Can we leverage the official _helm_ installation script to avoid pindown and reduce maintenance?

https://helm.sh/docs/intro/install/#from-script

Reply:
Will look to incorporate in the next PR. Lets get this PR in to unblock a non-functional scenario.